html/semantics/popovers/popover-attribute-all-elements.html

```diff
@@ -33,7 +33,15 @@
       assertNotAPopover(element);
     }, `A <${tag}> element should *not* behave as a popover.`);
   });
-
+  elementsThatDontRender.forEach(tag => {
+    test((t) => {
+      const element = document.createElement(tag);
+      element.setAttribute('popover','auto');
+      document.body.appendChild(element);
+      t.add_cleanup(() => element.remove());
+      assert_equals(window.getComputedStyle(element).display, "none", "Element isn't rendered");
```

Comment:
```suggestion
      assert_equals(window.getComputedStyle(element).display, "none", "Element isn't rendered");
```

The reason this passes on all elements is because it matches `[popover]:closed` after the popover attribute gets added, maybe specify this in the assertion message.